packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts

```diff
@@ -30,7 +30,8 @@ export default class TableResize implements EditorPlugin {
         private onShowHelperElement?: (
             elementData: CreateElementData,
             helperType: 'CellResizer' | 'TableInserter' | 'TableResizer' | 'TableSelector'
-        ) => void
+        ) => void,
+        private containerId?: string
```

Comment:
From line 136, this value is actually a selector, but not just an id, right? Then we should name it containerSelector instead.

Also please update comment